README.md

```diff
@@ -7,25 +7,51 @@ Latest build: https://metamask.github.io/api-specs/latest/openrpc.json
 
 JSON-RPC API provided by MetaMask.
 
-MetaMask dapp developers lack a formal RPC specification. This attempts to document which methods MetaMask supports in a way that's versionable, human-readable and machine-readable. It will improve the accuracy of documentation, API, and clients.
+MetaMask dapp developers lack a formal RPC specification. This attempts to document which methods MetaMask supports in a way that's versionable, human- and machine-readable. It will improve the accuracy of documentation, API, and clients.
 
-MetaMasks effectiveness as an enterprise-grade application platform depends on its ability to provide a reliable and predictable developer experience.
+MetaMask's effectiveness as an enterprise-grade application platform depends on its ability to provide a reliable and predictable developer experience.
 
 ## Contributing
 
+> Before making any significant changes to the API spec, consider whether to first submit a [MIP](https://github.com/MetaMask/metamask-improvement-proposals/tree/main) and complete the MIP [workflow](https://github.com/MetaMask/metamask-improvement-proposals/blob/main/PROCESS-GUIDE.md).
+
 Edit the `openrpc.yaml` file.
 
-You can then run `yarn run build` to generate the output file again.
+Run `yarn build` to regenerate the output file: `dist/build/openrpc.json`.
 
-To test the result, you can paste the resulting file from `dist/build/openrpc.json` into [the OpenRPC playground](https://playground.open-rpc.org/) and view the result.
+> To view the result, paste that file's contents into [the OpenRPC playground](https://playground.open-rpc.org/).
 
 ### Build Process
-- filter out methods that are not supported/implemented by metamask
-- merge wallet methods with execution api methods
-- output to `build`
 
+To build, run:
+
+- `yarn install`, if you haven't setup the repo
+- `yarn build`
```

Comment:
Remove, to reduce redundancy with contributing steps.

Reply:
moved/merged to Contributing